.gitmodules

```diff
@@ -113,3 +113,6 @@
 [submodule "tools/wgsim/src"]
 	path = tools/wgsim/src
 	url = https://github.com/lh3/wgsim.git
+[submodule "tools/hyphy/src"]
```

Comment:
Nitpick: Could you move this block up after `htslib` so this file is alphabetically sorted 🙂